### PR TITLE
final

### PR DESCRIPTION
**Pull request by YellowPipe**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+<body>
+	<header>
+		<div id = "youtubeTitle">
+			  <h1>Youtube</h1>
+		</div>
+		
+		<div id='searchbar'>
+		     <input type="text" name="search" placeholder="Search" size = "50">
+		     <button>Search</button>
+		<div>
```
Comment (heyalvaro) [inline index.html]: This should be a closing tag `</div>`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	</header>
+	<main>
+		<div id='video'>
+			<iframe width="560" height="315" src="https://www.youtube.com/embed/ZnuwB35GYMY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (heyalvaro) [inline index.html]: The frameborder attribute on the iframe element is obsolete. Use CSS instead.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+header {
+	width  : 100%;
+	height :100px;
+	display: flex;
+	flex-direction: row;
+	border-bottom: 2px solid grey;
+}
+#youtubeTitle{
```
Comment (heyalvaro) [inline style.css]: Why having a duplicate selector? Why not grouping with the selector in line 5?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+	padding: 10px;
+}
+#searchbar {
+	margin-left: 20%;
+	margin-right: 20%;
+	border-color: grey;
+}
+#searchbar input {
+	margin : 30px 0px 50px 100px ;
```
Comment (heyalvaro) [inline style.css]: If you are setting the value as `0` you can omit the units

File: index.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Youtube</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+    
+</head>
+<body>
+	<header>
+		<div id = "youtubeTitle">
```
Comment (heyalvaro) [inline index.html]: No logo?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Youtube</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+    
+</head>
+<body>
+	<header>
```
Comment (heyalvaro) [inline index.html]: What about having some icons at the end of the bar?

**Comment by rvvergara** [comment]
Hi Zina and Filipp,

Congratulations for your first project! :) Please find below some minor improvements you can make:

1. README - suggested contents: a) Short description of the project b)Names and links to github profiles c) Live demo (use raw.githack.com)

2. Please structure the files into separate folders. For instance all images should be in a 'image' folder, all css should be in a separate styles folder.

3. Great work on index.html (passed html validator). I won't suggest any more changes here as Alvaro already mentioned the needed improvements.

4. Nice work on the CSS! One thing, though; you used camel-case for naming classes (example, youtubeTitle). The better way to do this is use 'kebab' (so instead of youtubeTitle, it'd be youtube-title). This is the standard practice of naming css classes and id's. Other than that, your css is fine. 

5. Great that you commit often. One improvement moving forward is writing good commit messages. I myself am still striving for it but it's a good practice for all of us. A good commit message will describe exactly what was done in that particular commit (helps code reviews and also anybody collaborating with you or cloning/forking your project). You will get good at it :)

**Pull request merged**